base/essentials.jl

```diff
@@ -158,6 +158,19 @@ macro boundscheck(blk)
       $(Expr(:boundscheck,:pop)))
 end
 
+"""
+    @inbounds(blk)
+
+In the example below the bound check of array A is skipped to improve performance.
+
+    function sum(A::AbstractArray)
```

Comment:
This code block won't be syntax highlighted since it's not a fenced code block, rather use triple backticks with the language set to `julia` instead of an indented block.